freqtrade/freqai/freqai_interface.py

```diff
@@ -138,7 +138,7 @@ def start(self, dataframe: DataFrame, metadata: dict, strategy: IStrategy) -> Da
             dk = self.start_backtesting(dataframe, metadata, self.dk)
 
         dataframe = dk.remove_features_from_df(dk.return_dataframe)
-        self.clean_up()
+        # self.clean_up()
```

Comment:
Coming back to the spice rack for more spices requires that we don't force delete the data kitchen after the first spice is extracted. 

`self.dk/model` gets replaced automatically anyways next time the strategy comes back to access spice rack when `spice_rack_open == False`.

Reply:
it's fine if we aim to not clean up - i'm just raising a question if this can become a problem.
Usually when you have a "cleanup" call - it's to clean up (avoid memory leaks) - if that's not the case, no worries

Reply:
No this clean_up call is demonstrative only. I'll mark resolved.